src/components/tag-input/tag-input.component.html

```diff
@@ -5,6 +5,7 @@
     (click)="toggle()">
 
     <li *ngFor="let tag of _tags; let i = index" class="ux-tag"
+        [class.ux-tag-empty]="!tag"
```

Comment:
This class makes the delete button a bit wonky when compared to normal tags.

I do not believe that teams will actually use an empty string as the display value, since this would be very confusing to the user in both single and multiple select modes. I expect it will rather be something like this:
https://plnkr.co/edit/vSnRsqRFARUd9kBT

So no special handling is required for "empty tags" on our end, in my opinion.